lib/render/user.php

```diff
@@ -24,14 +24,20 @@ function userAvatar(
     });
 
     if (!$user) {
+        $userSanitized = $username;
+        sanitize_outputs($userSanitized);
+
+        $iconLabel = '';
         if ($icon !== false && ($icon || !$label)) {
-            return '';
+            $iconLabel = "<img src='" . media_asset('/UserPic/_User.png') . "' alt='$userSanitized' width='$iconSize' height='$iconSize'>";
         }
 
-        $userSanitized = $username;
-        sanitize_outputs($userSanitized);
+        $usernameLabel = '';
+        if ($label !== false && ($label || !$icon)) {
+            $usernameLabel = "<del>$userSanitized</del>";
+        }
 
-        return '<del>' . $userSanitized . '</del>';
+        return "<span class='inline whitespace-nowrap'>" . $iconLabel . $usernameLabel . "</span>";
```

Comment:
Not sure where the extra padding is coming from, but the images aren't aligned.
![image](https://user-images.githubusercontent.com/32680403/193368912-a6076975-a2e6-42fa-81df-4f3921634271.png)

Reply:
Thanks, that didn't occur to me.